content/applications/websites/website.rst

```diff
@@ -1,14 +1,51 @@
 :nosearch:
 :show-content:
 :hide-page-toc:
-:show-toc:
+
 
 =======
 Website
 =======
 
-Discover the best **Open-Source Website Builder** and learn how to build beautiful websites that
-convert visitors into leads or revenues.
+Odoo provides a user-friendly website builder that empowers you to create and manage your online
+presence effectively. Enhance lead generation and conversions effortlessly while ensuring seamless
+maintenance. Odoo covers all your needs from web design to comprehensive reporting and
+:abbr:`SEO (Search Engine Optimization)`.
+
+.. cards::
+
+   .. card:: Webdesign
+      :target: website/web_design
+      :large:
+
+      Odoo's editing tools help tailor your website to fit your business needs. Select from a wide
+      selection of pre-designed templates, building blocks, themes, color schemes, fonts, and more.
+
+
+   .. card:: Pages
+      :target: website/pages
+      :large:
+
+      Customize your website's structure by fine-tuning menus and adding pages. Boost your SEO
+      performance for improved ranking.
+
+   .. card:: Configuration
+      :target: website/configuration
+      :large:
+
+      Access a vast collection of high-quality, royalty-free images from our 3-million-strong
+      Unsplash database. Additionally, set up Google Search Console, your cookies bar, manage
+      translations, and handle multi-websites functionalities.
+
+   .. card:: Reporting
+      :target: website/reporting
+
+      Efficiently manage website analytics and integrate link trackers for comprehensive insights.
+
+   .. card:: Mail groups
+      :target: website/mail_groups
+
+      Easily configure mail groups for effective communication.
 
 .. important::
```

Comment:
I know you didn't change it but I don't think it needs to be an "important" admonition. It could be a note or a tip instead?